tests/xgboost/test_xgboost_sklearn_autolog.py

```diff
@@ -0,0 +1,498 @@
+from packaging.version import Version
```

Comment:
I think adding a simple test like below in `tests/xgboost/test_xgboost_autolog.py` should be enough.

```python

@pytest.mark.large
def test_xgb_autolog_sklearn():

    mlflow.xgboost.autolog()

    X, y = datasets.load_iris(return_X_y=True)
    params = {"n_estimators": 10, "reg_lambda": 1}
    model = xgb.XGBRegressor(**params)

    with mlflow.start_run() as run:
        model.fit(X, y)
        model_uri = mlflow.get_artifact_uri("model")

    client = mlflow.tracking.MlflowClient()
    run = mlflow.tracking.MlflowClient().get_run(run.info.run_id)
    assert run.data.metrics.items() <= params.items()
    artifacts = set(x.path for x in client.list_artifacts(run.info.run_id))
    assert artifacts >= set(["feature_importance_weight.png", "feature_importance_weight.json"])
    loaded_model = mlflow.xgboost.load_model(model_uri)
    np.testing.assert_allclose(loaded_model.predict(X), model.predict(X))
```